packages/api/src/index.ts

```diff
@@ -10,6 +10,16 @@
  * ```typescript
  * import { event, window, path } from '@tauri-apps/api'
  * ```
+ *
+ * ### Vanilla JS API
+ *
+ * The above import syntax is for TypeScript. If you're using vanilla JavaScript, use `window.__TAURI__` instead. It requires `app.withGlobalTauri` configuration option enabled.
```

Comment:
I think it would be more accurate to say JavaScript with bundler, what do you think about
```suggestion
 * The above import syntax is for JavaScript/TypeScript with a bundler. If you're using vanilla JavaScript, you can use the global `window.__TAURI__` object instead. It requires `app.withGlobalTauri` configuration option enabled.
```